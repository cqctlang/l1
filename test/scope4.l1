@global foo;
{
@define fool()
{
	printf("okay\n");
}

@define foo()
{
	fool();
}
}
foo();
