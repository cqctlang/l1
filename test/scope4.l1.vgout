scope4.l1:3: warning: assignment to unbound variable: fool
entry: var params, 2 locals, 0 temps
fool: 0 params, 0 locals, 0 temps
foo: 0 params, 0 locals, 0 temps
okay
