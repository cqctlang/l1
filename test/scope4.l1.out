scope4.l1:3: warning: assignment to unbound variable: fool
okay
